app/models/bot/smooch.rb

```diff
@@ -58,36 +58,26 @@ def suggestion_accepted?
       self.relationship_type_before_last_save.to_json == Relationship.suggested_type.to_json && self.is_confirmed?
     end
 
-    def self.inherit_status_and_send_report(rid)
+    def self.smooch_send_report(rid)
       relationship = Relationship.find_by_id(rid)
       unless relationship.nil?
         target = relationship.target
-        parent = relationship.source
-
         # Always inherit status for confirmed matches
```

Comment:
If you're moving the status inheritance part somewhere else, please update this comment accordingly.